boards/x86/ehl_crb/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ if(CONFIG_BOARD_EHL_CRB)
 set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
   COMMAND ${PYTHON_EXECUTABLE} ${PROJECT_SOURCE_DIR}/arch/x86/zefi/zefi.py
   -c ${CMAKE_C_COMPILER}
-  -o ${CMAKE_OBJCOPY}
+  -o objcopy
```

Comment:
will this now use host objcopy? will this work across architectures?

Reply:
No, its a stop gap until the SDK fix goes in.

I've got a SDK enhancement to enable EFI binary generation in the x86_64 toolchain.  So this is a temp fix, until we put out SDK 0.12.2 which I will do once we get a test SDK build and can confirm it works.